skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -359,6 +359,11 @@ cdef inline float _integral_to_distance_2d(IMGDTYPE [:, ::] integral,
     """
     References
     ----------
+    J. Darbon, A. Cunha, T.F. Chan, S. Osher, and G.J. Jensen, Fast
+    nonlocal filtering applied to electron cryomicroscopy, in 5th IEEE
+    International Symposium on Biomedical Imaging: From Nano to Macro,
+    2008, pp. 1331–1334.
```

Comment:
Some sneaky unicode in the endash!  Please define an encoding at the top of the file (or replace with '-').

This is what is causing the error reported by Travis.